07exponent.js

```diff
@@ -0,0 +1,30 @@
+// 7. Compute the exponent of a number.
+// The exponent of a number says how many times the base number is used as a factor.
+// 8^2 = 8 x 8 = 64. Here, 8 is the base and 2 is the exponent.
+
+//input: two integer numbers, the first number represents the base and the second represents the exponent
+//output: integer representing base^power
+//edge cases: negatives, base zero = 0 and power of zero = 1 
+//constraints: none
+
+/*
+Transformation steps:
+to the first power = return just the number
+create a total
+while exponent > 1 multiply total * base
+decrement exponent
+return total
+*/
+
+var total = 1;
```

Comment:
For many of the recursion you did, the variables can be from within the function.